testing/simapp/app.go

```diff
@@ -648,6 +650,12 @@ func (app *SimApp) LoadHeight(height int64) error {
 func (app *SimApp) ModuleAccountAddrs() map[string]bool {
 	modAccAddrs := make(map[string]bool)
 	for acc := range maccPerms {
+		// do not add mock module to blocked addresses
```

Comment:
This is required so that we can use mock module as a non blocked module account addr

Reply:
Nice we should add a changelog for this!